src/components/NewWorkspaceModal.js

```diff
@@ -91,6 +92,11 @@ const NewWorkspaceModal = withDisplayName('NewWorkspaceModal', ({
     }
   })
 
+  const getRawlsVersion = async () => {
```

Comment:
I am somewhat tempted to just put this in `ajax-common` itself.

Reply:
if it informs your opinion, we'll be able to delete this code in a month or so, once we're stably auto-deploying WDS from Rawls.

Reply:
Right, but are you going to delete from both places?